util/fsfat_test.h

```diff
@@ -0,0 +1,74 @@
+/** @file fsfat_test.h
+ *
+ * mbed Microcontroller Library
+ * Copyright (c) 2006-2016 ARM Limited
```

Comment:
this should be 2019 (new file or at least 2018-2019)

New files should contain SPDX identifier (look at platform header files).